recipe/meta.yaml

```diff
@@ -1,5 +1,7 @@
 {% set name = "plumed" %}
 {% set version = "2.7.1" %}
+{% set build = 1 %}
+{% set mpi = mpi or "nompi" %}
```

Comment:
@awvwgk Can you explain this line? Is mpi set to mpi or to nompi?

Reply:
This is only for the linter which usually does check the meta.yaml without first expanding the build matrix (here variants of mpi) defined in the conda_build_config.yaml